external/buildscripts/build.pl

```diff
@@ -1646,18 +1646,30 @@
 
 			system("ln", "-f", "$monoroot/mono/mini/mono-sgen","$distDirArchBin/mono") eq 0 or die("failed symlinking mono executable\n");
 			system("ln", "-f", "$monoroot/tools/pedump/pedump","$distDirArchBin/pedump") eq 0 or die("failed symlinking pedump executable\n");
+			if($ENV{MONODIS})
+			{
+				system("ln", "-f", "$monoroot/mono/dis/monodis","$distDirArchBin/monodis") eq 0 or die("failed symlinking monodis executable\n");
+			}
 			system('cp', "$monoroot/data/config","$distDirArchEtc/mono/config") eq 0 or die("failed to copy config\n");
 		}
 		elsif($^O eq 'darwin')
 		{
 			system("ln", "-f", "$monoroot/mono/mini/mono","$distDirArchBin/mono") eq 0 or die("failed hardlinking mono executable\n");
 			system("ln", "-f", "$monoroot/tools/pedump/pedump","$distDirArchBin/pedump") eq 0 or die("failed hardlinking pedump executable\n");
+			if($ENV{MONODIS})
+			{
+				system("ln", "-f", "$monoroot/mono/dis/monodis","$distDirArchBin/monodis") eq 0 or die("failed hardlinking monodis executable\n");
+			}
 		}
 		else
 		{
 			system("cp", "$monoprefix/bin/mono-2.0.dll", "$distDirArchBin/mono-2.0.dll") eq 0 or die ("failed copying mono-2.0.dll\n");
 			system("cp", "$monoprefix/bin/mono-2.0.pdb", "$distDirArchBin/mono-2.0.pdb") eq 0 or die ("failed copying mono-2.0.pdb\n");
 			system("cp", "$monoprefix/bin/mono.exe", "$distDirArchBin/mono.exe") eq 0 or die ("failed copying mono.exe\n");
+			if($ENV{MONODIS})
+			{
+				system("cp", "$monoprefix/bin/monodis.exe", "$distDirArchBin/monodis.exe") eq 0 or die ("failed copying monodis.exe\n");
```

Comment:
Does this do anything? As far as I can tell the windows executable is named something like `monodis-<gc>.exe`. I never verified that this line worked in my original implementation 😭

Reply:
Right, the chunk of code here for Windows will not really work because mono exe's and dll's on windows have mono-sgen/bdwgc suffix. For windows we now use `build_win_no_cygwin.pl`. 
I will remove it instead of adding to dead code.